content/en/docs/reference/kubectl/kubectl.md

```diff
@@ -369,6 +369,14 @@ kubectl [flags]
 </td>
 </tr>
 
+<tr>
+<td colspan="2">KUBECTL_PORT_FORWARD_WEBSOCKETS</td>
+</tr>
+<tr>
+<td></td><td style="line-height: 130%; word-wrap: break-word;">When set to true, the kubectl port-forward command will attempt to stream using the websockets protocol. If the upgrade to websockets fails, the commands will fallback to use the current SPDY protocol.
```

Comment:
How about:
```suggestion
<td></td><td style="line-height: 130%; word-wrap: break-word;">When set to true, the <tt>kubectl port-forward</tt> command attempts to stream using a <a href="https://www.w3.org/TR/websockets/">WebSocket</a>. If the upgrade to using a WebSocket fails, the kubectl tool falls back to using the <a href="https://www.chromium.org/spdy/">SPDY</a> protocol.
```

The suggested text avoids implying that SPDY is “current”. However, if this is different from the text already in k/k code, we should use that text from upstream as-is.

(it'd be great to get good docs into `kubectl` itself, where relevant)